packages/example-generator/src/run.ts

```diff
@@ -329,6 +329,21 @@ const UPDATE_PACKAGE_JSON: Mutator = {
         /"@osdk\/oauth": "\^.*?"/,
         `"@osdk/oauth": "workspace:*"`,
       )
+      .replace(
+        // Use locally generated SDK in the monorepo
+        /"@osdk\/widget-client-react.unstable": "\^.*?"/,
+        `"@osdk/widget-client-react.unstable": "workspace:*"`,
+      )
+      .replace(
+        // Use locally generated SDK in the monorepo
+        /"@osdk\/widget-client.unstable": "\^.*?"/,
+        `"@osdk/widget-client.unstable": "workspace:*"`,
+      )
+      .replace(
+        // Use locally generated SDK in the monorepo
+        /"@osdk\/widget.vite-plugin.unstable": "\^.*?"/,
+        `"@osdk/widget.vite-plugin.unstable": "workspace:*"`,
+      )
```

Comment:
@ssanjay1 I needed to introduce these replacements for the in-repo generator checks to pass